openedx/core/djangoapps/content/block_structure/config/__init__.py

```diff
@@ -28,8 +28,12 @@
 # .. toggle_name: block_structure.storage_backing_for_cache
 # .. toggle_implementation: WaffleSwitch
 # .. toggle_default: False
-# .. toggle_description: When enabled, stores block structures in a more permanent storage, like
-#   a database, which provides an additional backup for cache misses, instead of regenerating...
+# .. toggle_description: When enabled, block structures are stored in a more permanent storage,
+#   like a database, which provides an additional backup for cache misses, instead having them
+#   regenerated. The regenration of block structures is a time consuming processs. Therefore,
```

Comment:
processs => process

Reply:
I'm really sorry you're having to post such silly comments for my changes 😂 

My pull requests are usually much better, but the last couple of days have been hectic 😔 

Anyway, addressed 👍🏼